src/js/components/SidebarHeader.js

```diff
@@ -141,7 +141,24 @@ class SidebarHeader extends mixin(StoreMixin) {
     ];
 
     return (
-      <header className="header flex-item-shrink-0">
+      <header className="header">
+        <a className="header-dropdown">
+          <div className="header-content">
+            <div className="header-image-wrapper">
+              <div className="header-image" />
+            </div>
+            <div className="header-details">
+              <span className="header-title">
+                <span>
+                  {this.getClusterName()}
+                </span>
+              </span>
+              <span className="header-subtitle">
+                Firstname Lastname
```

Comment:
@ashenden is this a placeholder?

Reply:
@MatApple yes.  See my comments above about issue DCOS-18942. I'm currently using placeholder markup and content here that matches the preferred structure for the header and have commented out the old header.

Reply:
Also, I'm still working on resolving the tests that are failing.

Reply:
Hey @ashenden! Sorry for takin that long, why cant we adjust `UserAccountDropdownTrigger` component? Isn't there everything in place already?